core/src/com/mygdx/game/MyGdxGame.java

```diff
@@ -36,6 +40,14 @@ public class MyGdxGame extends Game {
 
   @Override
   public void create() {
+    CommonWebSockets.initiate();
```

Comment:
W sensie, Vertx jest po to, żeby pokazać, że kod libgdxowych websocketów jest kompatybilny z innymi websocketami?

Bo ogólnie to ten serwer to nadal będzie gra, tylko troch inna i nie wiem czy potrzeba czegoś takiego jak Vertx. Ale jak działa to ok. Kiedyś robiłem w tym jakąś stronę internetową, ale baaardzo dawno temu.

Reply:
Wyjaśnione wyżej